### PR TITLE
Loadbalancer controller

### PR DESCRIPTION
**Pull request by mqliang**
@ddysher PTAL

### CODE REVIEW

**Comment by mqliang** [comment]
@ddysher comments added. PTAL

File: loadbalancer-controller/api/helper.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package api
+
+
+import (
+	"k8s.io/client-go/1.5/pkg/runtime"
```
Comment (ddysher) [inline loadbalancer-controller/api/helper.go]: update vendor?

File: loadbalancer-controller/main.go [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+				{
+					Port:       int32(80),
+					TargetPort: intstr.FromInt(80),
+					Protocol:   v1.ProtocolTCP,
+				},
+			},
+		},
+	}
+
+	if _, err := clientset.Core().Pods("default").Create(&pod); err != nil && !errors.IsAlreadyExists(err) {
```
Comment (ddysher) [inline loadbalancer-controller/main.go]: ditto here

File: loadbalancer-controller/main.go [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+}
+
+// when nginx loadbalancer receive a request, and can not matching any ingress rules,
+// it forward it to default backend service. default-http-backend will always response
+// "404"
+func ensureDefaulBackendService(clientset *kubernetes.Clientset) error {
+	pod := v1.Pod{
+		ObjectMeta: v1.ObjectMeta{
+			Namespace: "default",
```
Comment (ddysher) [inline loadbalancer-controller/main.go]: Make namespace configurable; default is not good enough.

**Comment by ddysher** [comment]
Discussed offline.  I'll merge then work through the code base.

File: loadbalancer-controller/main.go (hunk 3 above)
Comment (ScorpioCPH) [inline loadbalancer-controller/main.go]: s/ensureDefaulBackendService/ensureDefaultBackendService

**Pull request merged**